packages/@ember/-internals/runtime/lib/system/core_object.ts

```diff
@@ -131,7 +167,7 @@ function initialize(obj, properties) {
     }
   }
 
-  sendEvent(obj, 'init', undefined, undefined, undefined, m);
+  sendEvent(obj, 'init', undefined, undefined, m);
```

Comment:
Was this just a straight-up bug happening with this `sendEvent()` before? 😬

Reply:
I believe so